<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAccountDialog</class>
 <widget class="QDialog" name="EditAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Librarian Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="first_name_line_dit"/>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="last_name_line_dit"/>
     </item>
     <item row="2" column="0" alignment="Qt::AlignTop">
      <widget class="QLabel" name="label_birth_date">
       <property name="text">
        <string>Birth Date</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_phone_number">
       <property name="text">
        <string>Phone Number</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" alignment="Qt::AlignTop">
      <widget class="QLabel" name="label_home_address">
       <property name="text">
        <string>Home Address</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_last_name">
       <property name="text">
        <string>Last Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="phone_number_line_dit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="email_line_dit"/>
     </item>
     <item row="2" column="1" alignment="Qt::AlignTop">
      <widget class="QDateEdit" name="birth_date_spin_box"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_first_name">
       <property name="text">
        <string>First Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_e_mail">
       <property name="text">
        <string>E-mail</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QTextEdit" name="home_address_text_dit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="new_password_line_dit"/>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="confirm_new_password_line_dit"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_confirm_new_password">
       <property name="text">
        <string>Confirm New Password</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_new_password">
       <property name="text">
        <string>New Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QLabel" name="label_notice">
       <property name="text">
        <string>(leave empty if you don't want to change your password)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="msg_label">
       <property name="text">
        <string>Your account info</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_button">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submit_button">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>first_name_line_dit</tabstop>
  <tabstop>last_name_line_dit</tabstop>
  <tabstop>email_line_dit</tabstop>
  <tabstop>phone_number_line_dit</tabstop>
  <tabstop>birth_date_spin_box</tabstop>
  <tabstop>home_address_text_dit</tabstop>
  <tabstop>new_password_line_dit</tabstop>
  <tabstop>confirm_new_password_line_dit</tabstop>
  <tabstop>close_button</tabstop>
  <tabstop>submit_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
